data/org.O1.swupdd.conf

```diff
@@ -0,0 +1,23 @@
+<?xml version="1.0" encoding="UTF-8"?> <!-- -*- XML -*- -->
+
+<!DOCTYPE busconfig PUBLIC
+ "-//freedesktop//DTD D-BUS Bus Configuration 1.0//EN"
+ "http://www.freedesktop.org/standards/dbus/1.0/busconfig.dtd">
+<busconfig>
+
+  <!-- This configuration file specifies the required security policies
+       for the swupd to work. -->
+
+  <!-- Only user root can own the fwupd service -->
+  <policy user="root">
+    <allow own="org.O1.swupdd.Client"/>
+  </policy>
+
+ <!-- Allow anyone to call into the service - we'll reject callers using PolicyKit -->
```

Comment:
is this the case? If not, remove the comment and I'd allow only user root and maybe wheel/admin group